src/graphql/mirror.js

```diff
@@ -353,6 +353,58 @@ export class Mirror {
       addConnection.run({id, fieldname});
     }
   }
+
+  /**
+   * Find objects and connections that are not known to be up-to-date.
+   *
+   * An object is up-to-date if its own data has been loaded at least as
+   * recently as the provided date.
+   *
+   * A connection is up-to-date if it has been fetched at least as
+   * recently as the provided date, and at the time of fetching there
+   * were no more pages.
+   */
+  _findOutdated(since: Date): QueryPlan {
+    const db = this._db;
+    return _inTransaction(db, () => {
+      const objects: $PropertyType<QueryPlan, "objects"> = db
+        .prepare(
+          dedent`\
+            SELECT typename AS typename, id AS id
```

Comment:
Re: `typename AS typename`, see:
  - <https://sqlite.org/c3ref/column_name.html> ("The name of a result column is the value of the "AS" clause for that column, if there is an AS clause. If there is no AS clause then the name of the column is unspecified and may change from one release of SQLite to the next.")
  - <https://github.com/JoshuaWise/better-sqlite3/issues/167>